pom.xml

```diff
@@ -188,11 +190,11 @@
       <scope>test</scope>
     </dependency>
 
-    <!-- Used actually in tests only, but if in test scope the transitive dep will be gone -->
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-classworlds</artifactId>
-      <version>2.5.2</version>
+      <version>2.6.0</version>
+      <scope>compile</scope>
```

Comment:
Do we really need this in compile?

Reply:
Comment is totally wrong, and it WAS in compile scope already, I just made it obvious.